scipy/stats/tests/test_discrete_distns.py

```diff
@@ -29,6 +29,34 @@ def test_hypergeom_logpmf():
     assert_almost_equal(hypergeom_logpmf, bernoulli_logpmf, decimal=12)
 
 
+def test_nhypergeom_logpmf():
+    # test with hypergeom
+    N, K, r = 45, 13, 8
+    k = 6
+    NHG = nhypergeom.pmf(k, N, K, r)
+    HG = hypergeom.pmf(k, N, K, k+r-1) * (N - K - (r-1)) / (N - (k+r-1))
+    assert_allclose(HG, NHG, rtol=10)
```

Comment:
`rtol` is the relative tolerance. It should be a very *small* number, e.g. `1e-10`.
```suggestion
    assert_allclose(HG, NHG, rtol=1e-10)
```